### PR TITLE
Cypress added

### PR DESCRIPTION
**Pull request by lobola**

### CODE REVIEW

File: cypress_e2e/Tests/home_page.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const { expect } = require("chai");
+
+describe("Loading home page", function(){
+    it("connects to index.html", function(){
+        cy.visit("http://localhost:7999");
+        cy.url().should('include', '/');
+    });
+});
+
```
Comment (matthewmcpolinkainos) [inline cypress_e2e/Tests/home_page.js:9]: empty lines 9 to 11 can be deleted

**Review by matthewmcpolinkainos — DISMISSED** [review]

**Review by felixmoore — DISMISSED** [review]
Looking good :) 👍

File: cypress_e2e/Tests/home_page.js (hunk 1 above)
Comment (lobola) [reply cypress_e2e/Tests/home_page.js:9]: Done

**Review by walkinator16000 — APPROVED** [review]
Looks good 😀

**Review by felixmoore — APPROVED** [review]

**Pull request merged**